tests/profiles-enabled/shipments/test_shipments_pytest.py

```diff
@@ -85,17 +85,11 @@ def test_successful_quickadd(self, client_alice, mock_aftership_create_success,
 
     def test_quickadd_create_fail(self, client_alice, mock_aftership_create_fail, assertions_create_tracking):
         response = client_alice.post(self.create_url, self.base_create_attributes)
-        AssertionHelper.HTTP_400(response, error='Aftership tracking supplied is already in use')
+        AssertionHelper.HTTP_400(response, error='Supplied tracking number has already been imported into Aftership')
         mock_aftership_create_fail.assert_calls(assertions_create_tracking)
 
     def test_quickadd_validation_fail(self, client_alice, mock_aftership_validation_failure,
                                       assertions_aftership_validation):
         response = client_alice.post(self.create_url, self.base_create_attributes)
-        AssertionHelper.HTTP_400(response, error='Invalid aftership_tracking supplied')
+        AssertionHelper.HTTP_400(response, error='Invalid aftership_tracking value')
         mock_aftership_validation_failure.assert_calls(assertions_aftership_validation)
-
-    def test_sqs_fail(self, client_alice, mock_aftership_create_success, assertions_create_tracking):
```

Comment:
I don't understand why this test was removed. We should ensure that if it fails to add it to the SNS topic, then the shipment isn't created. Otherwise we should have a retry option for publishing the SNS topic

Reply:
If the SNS topic doesn't exist (which was the only error you were catching), that's a configuration error, not a code error, so it doesn't add much value to unit test this. We already have SNS topics deployed for all of our environments.

As-is, if there is an unhandled exception with the SNS publication, it will bubble up and get converted into a 500 error by DRF and the Shipment will not be created.